### PR TITLE
Add "propagateSwipe" prop to allow scrolling inside modal

### LINKED ISSUE #236: ScrollView inside swipeable modal is not working
Just recently did an update and have confirmed that v7 stops the ability to scroll inside the modal whereas v6 was still working (swapped between the versions)

I looked at the diff from 6.5.0 up to 7.0.2 and the only thing I can see that may effect the scroll view would be [this line](https://github.com/react-native-community/react-native-modal/compare/v6.5.0...master#diff-1fdf421c05c1140f6d71444ea2b27638R194). 

I've tried adding the `Touchable...` around the views as other closed issues have suggested to no luck.

I've been stumped on this for a couple days now and not sure what could be wrong.

The modal is something of this structure:

```
<Modal>
    <ScrollView horiztonal>
        <FlatList ... />
       ...
    </ScrollView>
</Modal>
```

Any idea what changes from v6.5.0 to v7.0.2 could be causing the scrolling to not work?

### PR DESCRIPTION
**Pull request by cjroth**
Thanks @mmazzarolo for figuring this out.

Fixes #236.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -196,7 +198,9 @@ class ReactNativeModal extends Component {
         // work correctly even when the modal has touchable buttons.
         // For reference:
         // https://github.com/react-native-community/react-native-modal/pull/197
-        return Math.abs(gestureState.dx) >= 4 || Math.abs(gestureState.dy) >= 4;
```
Comment (mmazzarolo) [inline src/index.js]: Mind adding a link to the PR here?

**Comment by mmazzarolo** [comment]
@cjroth published on [`v7.1.0-beta.1`](https://github.com/react-native-community/react-native-modal/releases/tag/v7.1.0-beta.1), willing to give it a shot? 
Thanks for the contribution!

**Comment by jkdf2** [comment]
Sad to say this does not work for me. I've got a vertical ScrollView inside a modal with `swipeDirection="down"`. The ScrollView will not scroll.

**Comment by mmazzarolo** [comment]
😢 
@jkdf2 thanks for reporting.

**Comment by jkdf2** [comment]
Sure. Sorry to be the bearer of bad news.

Also, I would like to suggest that #236 and #109 be meged together. They appear to be about the same issue. I don't think the title is correct for #236, as `v6.5.0` also does not allow vertical scrolling a ScrollView inside a modal with `swipeDirection="down"`.

**Comment by mmazzarolo** [comment]
> Also, I would like to suggest that #236 and #109 be meged together.
You can merge issues on GitHub? 😮 Or maybe you meant to align the titles/discussions?  

Thanks for the suggestions btw.

**Comment by jkdf2** [comment]
Yeah, sorry, I just mean close one, open the other, and link the two with a comment :)

**Pull request merged**